packages/pluggableWidgets/datagrid-web/src/components/InfiniteBody.tsx

```diff
@@ -6,20 +6,25 @@ interface InfiniteBodyProps {
     hasMoreItems: boolean;
     setPage?: (computePage: (prevPage: number) => number) => void;
     isInfinite: boolean;
+    style?: any;
```

Comment:
Maybe be more specific? At least `object`

Reply:
OOps

Reply:
Changed to CSSProperties